gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -50,6 +50,8 @@
 
 
 def main(args):
+    from gnomad.resources.grch38.gnomad import POPS
```

Comment:
I just figured it out, the POPS import is broken because of lines 194-195:
```
if n_subsets_use_subpops:
    POPS = POPS_STORED_AS_SUBPOPS
```
I think python is angry or confused with the reassignment of POPS. might be a hacky fix, but you could do something like
```
pops = POPS 
if n_subsets_use_subpops:
    pops = POPS_STORED_AS_SUBPOPS
```

Reply:
Thank you!